src/material/list/selection-list.ts

```diff
@@ -230,6 +244,23 @@ export class MatSelectionList
     this.disabled = isDisabled;
   }
 
+  /**
+   * Whether the *entire* selection list is disabled. When true, each list item is also disabled
+   * and each list item is removed from the tab order (has tabindex="-1").
+   */
+  @Input()
+  override get disabled(): boolean {
```

Comment:
Do we have to override the getter? Its not doing anything different than the super class

Reply:
IIRC I had to override both the getter and the setter, I couldn't override just the setter. I can double check this later to see if I really need to override the getter.

Reply:
I researched this, and the short answer is "yes". Typescript sees the getter and setter as a single property, so if you override only the setter and not the getter, it doesn't inherit the getter from the parent class. We can se that if I delete the getter, typescript gives this error:

```
Property 'disabled' in type 'MatSelectionList' is not assignable to the same property in base type 'MatListBase'.
  Type 'BooleanInput' is not assignable to type 'boolean'.
    Type 'undefined' is not assignable to type 'boolean'.ts(2416)
Property 'disabled' in type 'MatSelectionList' is not assignable to the same property in base type 'SelectionList'.
  Type 'BooleanInput' is not assignable to type 'boolean'.ts(2416)
```

It's saying that trying to access the `disabled` property returns `undefined` because it does not have a getter. Typescript has an open issue [A derived class is allowed be created without a parent's getter or setter. #47581](https://github.com/microsoft/TypeScript/issues/47581).

Reply:
Since this is kinda an obscure typescript/javascript feature, I added a code comment to explain. It took some time, but I added a comment with the most simple way to explain it that I could think of.

LMK if the comment I added looks good, and I can resolve this conversation.